### PR TITLE
Bundle Subscription Price Test 1 (AB)

### PR DESCRIPTION
**Pull request by JustinPinner**
## What does this change?
Adds an AB test for digital subs pricing. Valid for non-paying readers visiting the UK front on devices running at tablet breakpoint and above, and who haven't been through the test before.

## What is the value of this and can you measure success?
We use a thrasher to drive in-test readers to a landing page hosted in membership frontend. All analysis will be done from that point.

## Does this affect other platforms - Amp, Apps, etc?
No, it's web only.

## Screenshots
screen shot 2017-04-28 at 15 55 56

## Tested in CODE?
Yes.

### CODE REVIEW

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/bundle-digital-sub-price-test-1.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                    !cookies.getCookie('GU_DBPT1') &&
+                    !userFeatures.isPayingMember() &&
+                    detect.isBreakpoint({min: 'tablet'}) &&
+                    config.page.isFront &&
+                    config.page.pageId.toLowerCase() === "uk" &&
+                    config.page.edition.toUpperCase() === 'UK';
+        };
+
+        this.thrasherContainer = function() {
+            return document.querySelector('#membership-ab-thrasher');
```
Comment (Ap0c) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/bundle-digital-sub-price-test-1.js]: Might prefer `getElementById` here, it's quite a bit [faster](https://jsperf.com/getelementbyid-vs-queryselector).

Comment (JustinPinner) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/bundle-digital-sub-price-test-1.js]: 🙄 of course! I'd been doing so much with class selection, it didn't register that this was an ID match.

**Review by Ap0c — APPROVED** [review]
Looks good! ⛵️

Dropped into all three tests using (courtesy of @SiAdcock's excellent devtools), all seemed to work fine.

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/bundle-digital-sub-price-test-1.js (hunk 1 above)
Comment (JustinPinner) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/bundle-digital-sub-price-test-1.js]: I'm tempted to leave this as-is right now on the basis that it's how it was last time around. If we decide to adjust the audience or applicable fronts prior to activating the test, I'll change it then.

**Comment by JustinPinner** [comment]
Thanks @Ap0c

**Pull request merged**